bin/hs-airdrop

```diff
@@ -271,8 +271,14 @@ async function findNonces(key, priv) {
     }
   }
 
-  if (out.length === 0)
-    throw new Error(`Could not find nonce in bucket ${bucket}.`);
+  if (out.length === 0) {
```

Comment:
@pinheadmz I would prefer to stick with convention, i.e. throw an error. Is there a reason you use `console.log` + `process.exit` instead?

Reply:
To get multiple lines of output and also to not dump a useless stack trace on the user. I'm not sure this is the best approach or even the best language option, everything's open for discussion... just starting to get nervous about all the support requests...

Reply:
> To get multiple lines of output and also to not dump a useless stack trace on the user.

I understand.